go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-bindata/go-bindata v3.1.1+incompatible
+	github.com/go-openapi/spec v0.19.5
```

Comment:
this is surprising... is it correct?

Reply:
Modified by `hack/update-vendor.sh`, I had fixed it.